src/start_package/mod.rs

```diff
@@ -118,6 +119,16 @@ pub async fn execute(package_dir: &Path, url: &str) -> Result<()> {
     let zip_filename = target_dir.join(&pkg_publisher).with_extension("zip");
     zip_directory(&pkg_dir, &zip_filename.to_str().unwrap())?;
 
+
+    let mut file = fs::File::open(&zip_filename)?;
+    let mut hasher = Sha256::new();
+    let mut buffer = Vec::new();
+    file.read_to_end(&mut buffer)?;
+    hasher.update(&buffer);
+    let hash_result = hasher.finalize();
+    let hash_hex = format!("{:x}", hash_result);
```

Comment:
This should be rolled into the `println!()`/`info!()` below, if possible (I think the formatting should work there).